spotlight-client/src/RacialDisparitiesNarrativePage/RacialDisparitiesNarrativePage.tsx

```diff
@@ -94,39 +103,69 @@ const RacialDisparitiesNarrativePage: React.FC<RacialDisparitiesNarrativePagePro
               {narrative.isLoading || narrative.isLoading === undefined ? (
                 <Loading />
               ) : (
-                <NarrativeIntroCopy>
+                <IntroCopy>
                   {HTMLReactParser(pupa(narrative.introduction, templateData))}
-                </NarrativeIntroCopy>
+                </IntroCopy>
+              )}
+              {narrative.populationDataSeries && (
+                <BarChartPair
+                  data={narrative.populationDataSeries}
+                  download={() => narrative.downloadPopulation()}
+                  filters={[]}
+                  methodology={narrative.introductionMethodology}
+                />
               )}
             </NarrativeIntroContainer>
           ),
         },
-        ...narrative.sections.slice(0, -1).map((section) => {
-          return {
-            title: section.title,
-            contents: (
-              <StickySection
-                leftContents={
-                  <>
-                    <NarrativeSectionTitle>
-                      {section.title}
-                    </NarrativeSectionTitle>
-                    {narrative.isLoading ||
-                    narrative.isLoading === undefined ? (
+        ...narrative.sections.map((section) => {
+          // all sections except the conclusion should look like this
+          if ("chartData" in section) {
+            return {
+              title: section.title,
+              contents: (
+                <StickySection
+                  leftContents={
+                    <>
+                      <NarrativeSectionTitle>
+                        {section.title}
+                      </NarrativeSectionTitle>
+                      {narrative.isLoading ||
+                      narrative.isLoading === undefined ? (
```

Comment:
I'm curious why `isLoading` is initially unset instead of set to `true`, I think we may be setting it initially to true in lantern and I was wondering if you noticed unintended behavior with that?